server/src/internalClusterTest/java/org/elasticsearch/discovery/SnapshotDisruptionIT.java

```diff
@@ -228,6 +229,48 @@ public void testDisruptionAfterShardFinalization() throws Exception {
                 client(masterNode).admin().cluster().prepareCreateSnapshot(repoName, "snapshot-2").setWaitForCompletion(true).get();
         final SnapshotInfo successfulSnapshotInfo = successfulSnapshot.getSnapshotInfo();
         assertThat(successfulSnapshotInfo.state(), is(SnapshotState.SUCCESS));
+
+        logger.info("--> making sure snapshot delete works out cleanly");
+        assertAcked(client().admin().cluster().prepareDeleteSnapshot(repoName, "snapshot-2").get());
```

Comment:
Had to add this to force a cleanup run so the repo consistency check doesn't start complaining about leaked files from the broken snapshot run in this test.